geometry_msgs/PoseStamped indata
---
string[2] outmsg
